addons/website_blog/static/src/js/options.js

```diff
@@ -121,6 +121,11 @@ options.registry.BlogPostTagSelection = options.Class.extend({
             'display_name': widgetValue,
         };
         this.tagIDs.push(newTagID);
+        return new Promise(resolve => {
+            this.trigger_up('snippet_option_update', {
+                onSuccess: () => resolve(),
+            });
+        });
```

Comment:
- This has a huge need of comment
- Even if working this is probably way too hacky. Option methods are not supposed to trigger that by themselves nor care about UI updates...
- I don't actually understand it so I cannot suggest alternatives :grimacing: (Need more info, I understand the alphabetical order may be a cause of error but I don't see why in this case, should not the create method be called only anyway?)